monkey/monkey_island/cc/services/initialize.py

```diff
@@ -138,10 +138,14 @@ def _decorate_island_event_queue(
 
 
 def _register_repositories(container: DIContainer, data_dir: Path):
-    container.register_instance(
+    container.register_convention(
         IFileRepository,
+        "file_repository",
         _decorate_file_repository(LocalStorageFileRepository(data_dir / "runtime_data")),
     )
+    container.register_convention(
+        IFileRepository, "plugin_file_repository", LocalStorageFileRepository(data_dir / "plugins")
```

Comment:
Do we want to decorate this file repository?

Reply:
Definitelly, just missed the decorator call